.eslintrc.cjs

```diff
@@ -0,0 +1,21 @@
+require('@uniswap/eslint-config/load')
+
+module.exports = {
+  extends: ['@uniswap/eslint-config/node'],
+
+  overrides: [
+    {
+      // Somehow the import/no-unused-modules rule can't be turned off in rules
```

Comment:
Why turning this one off?

Reply:
It was giving some false-positives and keeps giving errors in the development, so thought maybe we should turn off until the next pr

Reply:
I have the same issue sometimes, restarting the server solve the issue most of the time, but I agree it's quite annoying. We can keep it turned off for the moment